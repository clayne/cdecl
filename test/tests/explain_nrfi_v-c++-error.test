EXPECTED_EXIT=65
cdecl -xc++ <<END
explain noreturn void f(int)
END
