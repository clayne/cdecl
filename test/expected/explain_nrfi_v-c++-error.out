explain noreturn void f(int)
        ^
9: error: "noreturn": unknown name; did you mean "co_return" or "return"?
